[openbsd] hide support column and update summary (#913)
captn3m0/endoflife.date: products/openbsd.md

> > [The OpenBSD project](https://www.openbsd.org/) produces a FREE, multi-platform 4.4BSD-based UNIX-like operating system. Our efforts emphasize portability, standardization, correctness, proactive security and integrated cryptography.

Reviewer: We shouldn't talk about openbsd [in first-person](https://github.com/endoflife-date/endoflife.date/wiki/Guiding-Principles#consistent-tone):

```suggestion
> [The OpenBSD project](https://www.openbsd.org/) produces a free, multi-platform 4.4BSD-based UNIX-like operating system. It emphasizes portability, standardization, correctness, proactive security and integrated cryptography.
```

Author: Updated.
